Makefile

```diff
@@ -6,8 +6,18 @@ GO_FILES := $(shell \
 	-o -name '*.go' -print | cut -b3-)
 
 .PHONY: bench
-bench:
-	go test -bench=. ./...
+bench: bin/benchstat bin/benchart
```

Comment:
Would you be open to putting up the benchmarking changes as a separate PR, to be landed before this one?

I'd love to keep the history clean/separate.

Reply:
Yup: https://github.com/uber-go/ratelimit/pull/86

Reply:
Nice. thanks. I'll wait for you to rebase.

Reply:
Rebased

Reply:
Wait, this is not rebased, right? I'm seeing a bunch of changes that are already on main.

I'm not intimately familiar with github workflows, but I'm assuming there's a way to:
- rebase your changes on top of main
- potentially squash the commit

I wonder if I can figure out how to do this myself.

Reply:
Hey, I squashed all commits into one.

Reply:
I don't understand github then - I expected it's possible to have this PR **ONLY** list the things we're actually changing here. I expected all the makefile, go.sum, etc changes to be gone/not visible.

Is that not possible? If so, do we need a new PR rebased on the changes already in `main`?

Reply:
Okay, I think I clicked the right button now.

Lemme take one last look later today, I'll merge it in after.